### PR TITLE
fixing an issue with the tool by updating output files copying condit…

### PR DESCRIPTION
**Pull request by EngyNasr**
fixing an issue with the tool by updating output files copying conditions, where user get no error when no databases are provided, based on that no output means no error rather no output

Thanks a lot,
Engy

### CODE REVIEW

**Comment by wm75** [comment]
I think this `| true` magic justifies an explanatory comment in the source code or future contributors will be very puzzled what this is about.

**Comment by wm75** [comment]
In addition, you need to bump the wrapper version.

**Comment by EngyNasr** [comment]
> In addition, you need to bump the wrapper version.

Can you please explain more for both comments please :)

**Comment by gallardoalba** [comment]
> > In addition, you need to bump the wrapper version.
> 
> Can you please explain more for both comments please :)

You need to increase the version suffix https://github.com/galaxyproject/tools-iuc/blob/38b55875aabf987c81372688d8c7462afefd263b/tools/srst2/macros.xml#L4

File: tools/srst2/srst2.xml [hunk 1]
```diff
@@ -97,17 +99,17 @@ srst2
     --keep_interim_alignment
 #end if
 #if 'report_new_consensus' in str($output_files_selector) and $use_gene_db.selector == "yes" and $use_mlst_db.selector == "yes"
-&& mkdir -p allelesOutput/ && cp *.output__input.*.pileup allelesOutput
+&& mkdir -p allelesOutput/ && cp *.output__input.*.pileup allelesOutput | true
 #end if
 #if $use_gene_db.selector == "yes" and  $use_gene_db.no_gene_details
-&& mkdir -p geneTypingOutput/ && cp output__genes__*__results.txt geneTypingOutput && cp output__fullgenes__*__results.txt geneTypingOutput
+&& mkdir -p geneTypingOutput/ && cp output__genes__*__results.txt geneTypingOutput | true && cp output__fullgenes__*__results.txt geneTypingOutput | true
```
Comment (gallardoalba) [inline tools/srst2/srst2.xml:105]: Also as Wolfgag suggested, it would be nice to explain why did you include the `| true` expressions.

**Review by gallardoalba — APPROVED** [review]

**Pull request merged**